guides/release/configuring-ember/disabling-prototype-extensions.md

```diff
@@ -76,15 +76,15 @@ required interfaces using the convenience method `Ember.A`:
 import { A } from '@ember/array';
 
 let islands = ['Oahu', 'Kauai'];
-islands.includes('Oahu');
-// => TypeError: Object Oahu,Kauai has no method 'includes'
+islands.without('Oahu');
```

Comment:
I'd rather not use `.without` as the replacement here. Can we use either `.pushObject` (there are more Hawaiian islands 😛) or swap the items to small objects and use something like `.findBy('isBigIsland', false)`?

My concern with `.without` is that it isn't **really** present due to anything with `@ember/array` it's more of a fundamental internal `@ember/object` thing, so using it as the example is a bit more confusing than something you _would_ expect to call on an `@ember/array`.